src/StreamJsonRpc.Shared/Reflection/TargetMethod.cs

```diff
@@ -173,12 +209,12 @@ private static object[] TryGetParameters(JsonRpcMessage request, MethodSignature
             }
         }
 
-        private bool TryAddMethod(JsonRpcMessage request, Dictionary<MethodSignature, object[]> targetMethods, MethodInfo method, JsonSerializer jsonSerializer)
+        private bool TryAddMethod(JsonRpcMessage request, Dictionary<MethodSignature, object[]> targetMethods, MethodInfo method, JsonSerializer jsonSerializer, string methodAttribute = null)
```

Comment:
The `methodAttribute` parameter name sounds like it's an attribute. But it's only a string -- the value of a property from the attribute. Can we give this a more descriptive name like `requestedMethodName` or `clrMethodName`?